reference/shipping.v2.yml

```diff
@@ -2337,6 +2369,12 @@ components:
           description: Whether or not this shipping method is a fallback method used when advanced shipping rules are unavailable.
           example: false
           type: boolean
+        channels:
+          description: List of channels associated to a method. When creating a new method, all available channels are associated by default.
+          example: [1, 3]
+          type: array
+          items:
+            type: integer
```

Comment:
Would it be beneficial to indicate that this is an optional field?

Reply:
Thanks mate, I've added `(optional)` to the description for both instances.🍻